kudu-handler/src/java/org/apache/hadoop/hive/kudu/KuduHiveUtils.java

```diff
@@ -104,11 +104,10 @@ public static void importCredentialsFromCurrentSubject(KuduClient client) {
         // 'client'. This is necessary if we want to support a job which
         // reads from one cluster and writes to another.
         if (!tok.getService().equals(service)) {
-          LOG.debug("Not importing credentials for service " + service +
-              "(expecting service " + service + ")");
+          LOG.debug("Not importing credentials for service {} (expecting service {})", service, service);
```

Comment:
Passing the same object two times does not make much sense but I guess outside the scope of the PR.